src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__cell calendar__cell--1"></div>
+      <div class="calendar__cell calendar__cell--2"></div>
```

Comment:
you shouldn't use modifiers for calendar__cell. Just add values with scss
```suggestion
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
```